package.json

```diff
@@ -5,12 +5,10 @@
   "main": "dist/app.js",
   "scripts": {
     "build": "tsc",
-    "start": "node src/app.ts",
```

Comment:
This is a common script for all projects. I prefer to have a start script in all projects. It is default script for running apps in prod mode